public/video-ui/src/reducers/uploadsReducer.js

```diff
@@ -0,0 +1,21 @@
+export default function upload(state = null, action) {
```

Comment:
Ah this was a case of copy-paste ignorance on my part. I've changed the default state to:

```javascript
{ complete: false }
```